.github/workflows/land-blocking.yml

```diff
@@ -11,19 +11,72 @@ jobs:
     name: Build images and run cluster test
     runs-on: self-hosted
     steps:
-    - uses: actions/checkout@v1
-    - name: Setup env
-      run: |
-        echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
-    - name: Build, tag and push images
-      run: |
-        docker/build-aws.sh --build-all --version $LIBRA_GIT_REV --addl_tags canary
-    - name: Launch cluster test
-      run: |
-        JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
-        ./scripts/cti \
-          --marker land \
-          --tag dev_$LIBRA_GIT_REV \
-          --report report.json \
-          --run bench \
-          -- --duration 60
+      - uses: actions/checkout@v1
+      - name: Setup env
+        run: |
+          echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
+      - name: Build, tag and push images
+        run: |
+          docker/build-aws.sh --build-all --version $LIBRA_GIT_REV --addl_tags canary
+      - name: Launch cluster test
+        run: |
+          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+          ./scripts/cti \
+            --marker land \
+            --tag dev_$LIBRA_GIT_REV \
+            --report report.json \
+            --run bench \
+            -- --duration 60
+      - name: Post test results on PR
+        if: always()
+        uses: actions/github-script@0.4.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            should_fail = false;
```

Comment:
Can we use `let`? So `let should_fail = false;`.

Reply:
Also, are we missing a `let body = "";`? Where is `body` initialized?

Reply:
Good call. Was following the github-script examples.  Will update.

Reply:
Fixed.